Cargo.toml

```diff
@@ -25,22 +31,32 @@ exclude = [
 ]
 keywords = ["uProtocol", "SDK", "vsomeip", "SOMEIP"]
 license = "Apache-2.0"
-name = "up-client-vsomeip-rust"
 readme = "README.md"
-repository = "https://github.com/eclipse-uprotocol/up-client-vsomeip-rust"
-rust-version = "1.72"
+repository = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust"
+rust-version = "1.76"
 version = "0.1.0"
 
-[dependencies]
+[workspace.dependencies]
 async-trait = { version = "0.1" }
+bimap = { version = "0.6.3" }
 bytes = { version = "1.5" }
+console-subscriber = { version = "0.3.0" }
+crossbeam-channel = { version = "0.5.13" }
+cxx = { version = "1.0" }
 rand = { version = "0.8" }
 regex = { version = "1.10" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
-up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", version = "0.1.5" }
+tokio = { version = "1.35.1", features = ["rt", "rt-multi-thread", "macros", "sync", "time", "tracing"] }
+up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", rev = "3a50104421a801d52e1d9c68979db54c013ce43d" }
 url = { version = "2.5" }
-uuid = { version = "1.7", features = ["v8"] }
+uuid = { version = "1.7", features = ["v8", "v4"] }
```

Comment:
do you really need this?

Reply:
Used it at one point, not anymore. Good catch.

Reply:
Removed, along with several other dependencies no longer in use.